src/js/base/stories/NoneFoundStories/NoneFound.stories.js

```diff
@@ -1,5 +1,5 @@
 import React from "react";
-import { NoneFound } from "../NoneFound";
+import { NoneFound } from "../../NoneFound";
 
```

Comment:
I don't think the "None" story here makes sense, as not passing a noun is a mistake and not an intended use case. If the goal is to show that the noun is inserted into the sentence then you could replace the story with a version that shows the replaced location by passing a string literal like "{plural noun}" into the `noun` parameter. (Making the final string "No {plural nouns} found")